configs/rotated_fcos/README.md

```diff
@@ -0,0 +1,50 @@
+# Rotated FCOS
+
+> [FCOS: Fully Convolutional One-Stage Object Detection](https://arxiv.org/abs/1904.01355)
+<!-- [ALGORITHM] -->
+
+## Abstract
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/40661020/143882011-45b234bc-d04b-4bbe-a822-94bec057ac86.png"/>
+</div>
+
+We propose a fully convolutional one-stage object detector (FCOS) to solve object detection in a per-pixel prediction
+fashion, analogue to semantic segmentation. Almost all state-of-the-art object detectors such as RetinaNet, SSD, YOLOv3,
+and Faster R-CNN rely on pre-defined anchor boxes. In contrast, our proposed detector FCOS is anchor box free, as well
+as proposal free. By eliminating the predefined set of anchor boxes, FCOS completely avoids the complicated computation
+related to anchor boxes such as calculating overlapping during training. More importantly, we also avoid all
+hyper-parameters related to anchor boxes, which are often very sensitive to the final detection performance. With the
+only post-processing non-maximum suppression (NMS), FCOS with ResNeXt-64x4d-101 achieves 44.7% in AP with single-model
+and single-scale testing, surpassing previous one-stage detectors with the advantage of being much simpler. For the
+first time, we demonstrate a much simpler and flexible detection framework achieving improved detection accuracy. We
+hope that the proposed FCOS framework can serve as a simple and strong alternative for many other instance-level tasks.
+
+## Results and Models
+
+DOTA1.0
+
+|         Backbone         |  mAP  | Angle | Separate Angle | Tricks | lr schd | Mem (GB) | Inf Time (fps) | Aug | Batch Size |                                                Configs                                                |          Download          |
+|:------------------------:|:-----:|:-----:|:--------------:|:------:|:-------:|:--------:|:--------------:|:---:|:----------:|:-----------------------------------------------------------------------------------------------------:|:--------------------------:|
+| ResNet50 (1024,1024,200) | 70.70 | le90  |       Y        |   Y    |   1x    |    -     |       -        |  -  |     2      |    [rotated_fcos_sep_angle_r50_fpn_1x_dota_le90](./rotated_fcos_sep_angle_r50_fpn_1x_dota_le90.py)    | [model](-) &#124; [log](-) |
```

Comment:
`Mem (GB)` is recorded in log.json file.
Inf Time (fps) can be test by:
`python -m torch.distributed.launch --nproc_per_node=1 --master_port=29500 tools/analysis_tools/benchmark.py YOUR_CFG YOUR_MODEL --launcher pytorch`